.gitignore

```diff
@@ -2,7 +2,9 @@
 doc/
 Gemfile.lock
 coverage/
-spec/rails_app/tmp/
-spec/rails_app/log/
+spec/rails_app/log/*
+spec/rails_app/tmp/*
+!spec/rails_app/log/.keep
+!spec/rails_app/tmp/.keep
```

Comment:
Could you remove these two lines and instead simply add the two files with `git add --force` instead?

Reply:
@fphilipe I didn't need to do anything other than remove those 2 lines since the files are already committed. I checked this branch on GitHub and both files are showing up so its good. Thanks!